torch/_inductor/codegen/wrapper.py

```diff
@@ -2541,10 +2541,16 @@ def _generate_kernel_call_helper(
         original_fxnode_name=None,
     ):
         device = device or V.graph.get_current_device_or_throw()
-        if not (
-            triton or device.type not in ("cpu", "mps")
-        ):  # TODO: Fix me, MPS does not expose streams now
-            self.writeline(self.wrap_kernel_call(kernel_name, call_args))
+        if not triton:
+            if device.type == "cpu":
+                self.writeline(self.wrap_kernel_call(kernel_name, call_args))
+            elif device.type == "mps":
+                # TODO: Fix me, MPS does not expose streams now
+                self.writeline(
+                    self.wrap_kernel_call(f"{kernel_name}.generated_kernel", call_args)
+                )
+            else:
```

Comment:
This seems unrelated to dynamic shapes is it?

Reply:
In the [above change](https://github.com/pytorch/pytorch/pull/159355/files/6160eec2d54546ecb02b5de225b77559e8e88e88#r2243650998) since I changed the kernel name from "mps_lib_0.generate_kernel" to be "mps_lib_0", when we call the kernel we now need to add the ".generated_kernel" so that the callsite is correct.